RELEASE-NOTES.rst

```diff
@@ -1,54 +1,15 @@
 ===============================
- Flask-IIIF v0.2.0 is released
+ Flask-IIIF v0.2.1 is released
 ===============================
 
-Flask-IIIF v0.2.0 was released on May 22, 2015.
+Flask-IIIF v0.2.1 was released on May 22, 2015.
 
 About
 -----
 
 Flask-IIIF is a Flask extension permitting easy integration with the
 International Image Interoperability Framework (IIIF) API standards.
 
-Incompatible changes
---------------------
-
-- Removes `uuid_to_path_handler` callback.
-
-- Updates error classes names (MultimediaImageResizeError and
-  MultimediaImageCropError).
-
-New features
-------------
-
-- Adds image information request endpoint `<uuid>/info.json` which
-  contains available metadata for the image, such as the full height
-  and width, and the functionality available for the image, such as
-  the formats in which it may be retrieved, and the IIIF profile used.
-
-- Adds new signals to REST API that permits to have access before and
-  after process of the request as well as after the validation of
-  IIIF.
-
-- Adds a configurable decorator to the REST API which can be configure
-  with the `api_decorator_handler`.
-
-- Adds the `uuid_to_image_opener_handler` which can handle both
-  `fullpath` and `bytestream` as source.
-
-Improved features
```

Comment:
Improved features
-----------------

- Fixes a problem where small images can trigger server errors when used with
  the IIIF Presentation API.
- Extracts resize resampling method as a config variable.
- Adds signals before and after image info requests.
- Adds `dl` request parameter to specify name of file for download
  (e.g. `..default.jpg?dl=image_3.jpg`)